OneBlink.SDK/ApprovalsClient.cs

```diff
@@ -34,7 +33,7 @@ public ApprovalsClient(string accessKey, string secretKey, TenantName tenantName
             List<string> lastUpdatedBy = null
         )
         {
-            NameValueCollection searchParams = System.Web.HttpUtility.ParseQueryString(string.Empty);
+            IDictionary<string, string> searchParams = new Dictionary<string, string>();

```

Comment:
Will this work for duplicated keys?  e.g. Multiple status's or updatedBy?

Reply:
great question, prob not.